packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -1,52 +1,27 @@
 /**
  * External dependencies
  */
-import { I18nManager } from 'react-native';
 import { first, last, partial, castArray } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { ToolbarButton } from '@wordpress/components';
-import { __, sprintf } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { withInstanceId, compose } from '@wordpress/compose';
-import { arrowUp, arrowDown, arrowLeft, arrowRight } from '@wordpress/icons';
 
-const horizontalMover = {
-	backwardButtonIcon: arrowLeft,
-	forwardButtonIcon: arrowRight,
-	backwardButtonHint: __( 'Double tap to move the block to the left' ),
-	forwardButtonHint: __( 'Double tap to move the block to the right' ),
-	firstBlockTitle: __( 'Move block left' ),
-	lastBlockTitle: __( 'Move block right' ),
-	/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
-	backwardButtonTitle: __(
-		'Move block left from position %1$s to position %2$s'
-	),
-	/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
-	forwardButtonTitle: __(
-		'Move block right from position %1$s to position %2$s'
-	),
-};
-
-const verticalMover = {
-	backwardButtonIcon: arrowUp,
-	forwardButtonIcon: arrowDown,
-	backwardButtonHint: __( 'Double tap to move the block up' ),
-	forwardButtonHint: __( 'Double tap to move the block down' ),
-	firstBlockTitle: __( 'Move block up' ),
-	lastBlockTitle: __( 'Move block down' ),
-	/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
-	backwardButtonTitle: __( 'Move block up from row %1$s to row %2$s' ),
-	/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
-	forwardButtonTitle: __( 'Move block down from row %1$s to row %2$s' ),
-};
+/**
+ * Internal dependencies
+ */
+import {
+	getMoverDescription,
+	getArrowIcon,
+	getMoverButtonTitle,
+} from './mover-description';
```

Comment:
I have prepared some refactor that might looks cleaner.

See [here](https://github.com/WordPress/gutenberg/pull/22217) and let me know WDYT